client/payment-details/dispute-details/dispute-due-by-date.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { dateI18n } from '@wordpress/date';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import classNames from 'classnames';
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import type { EvidenceDetails } from 'wcpay/types/disputes';
+
+const DisputeDueByDate: React.FC< {
```

Comment:
Nice clean single-purpose component. Do you think it needs docs to explain what it does, or does the component name make that clear? What does a future developer need to know to use this component? How tightly coupled to disputes is it – is it more general?